src/atoms/buttons/button.scss

```diff
@@ -1,3 +1,8 @@
+@import "../../assets/scss/0-vendor/__vendor";
```

Comment:
thought: consider creating a `_global` partial SCSS file with these imports, then you'd only need one `@import "../../_global";` line.

Reply:
Love the idea @mrjones2014. I added it in a later commit